src/apb/dat/Dockerfile.j2

```diff
@@ -1,5 +1,4 @@
 FROM ansibleplaybookbundle/apb-base
-# MAINTAINER {{ $MAINTAINER }}
 
 LABEL "com.redhat.apb.version"="0.1.0"
```

Comment:
This doesn't need to be addressed in this PR but I'm wondering if the version should be generated by the tooling for the Dockerfile. Technically we are up to 0.1.1 in PyPi and releases on Github I'm not sure if this version number should correlate to the tooling version number or just referring to the spec version of the APBs being published.

I'll discuss this with @cfchase and we'll address it later.